posts/2016-12-17-scala-coc.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: "Endorsing the new Scala Code of Conduct"
+
+meta:
+  nav: blog
+  author: typelevel
+---
+
+A couple of days ago, the new [Scala Code of Conduct](https://contributors.scala-lang.org/t/please-read-scala-code-of-conduct/28) has been published.
+It applies to all official Scala channels, including mailing lists, Gitter channels and GitHub repositories.
+We would like to take this opportunity to endorse this new Code of Conduct.
+From our perspective, it does a good job of listing encouraged behaviour, instead of just banning harassment: It reflects the goal of _actively_ creating a welcoming community.
+Also, we consider it to be a decent substitute of our own [Code of Conduct](/conduct.html).
+That means that Typelevel project maintainers are free to switch to the Scala Code of Conduct.
+At this point, there is no need to do so, but we would like to encourage Typelevel project maintainers to consider switching at their own discretion.
```

Comment:
I was trying to make the point that maintainers don't have to switch. Can you suggest alternative wording for that?